pep8.py

```diff
@@ -1846,7 +1846,15 @@ def check_files(self, paths=None):
         report.start()
         try:
             for path in paths:
-                if os.path.isdir(path):
+                if path == "-":
+                    # STDIN
+                    path = self.options.stdin_name
```

Comment:
If someone doesn't specify a name, then it defaults to the current behaviour. The name gets assigned to `'STDIN'`, or something similar (don't have it in front of me right now, but I manually tested that case).